src/assets/styles/themes/dark.less

```diff
@@ -154,6 +154,18 @@
   // Message
   --ergo-message-box-shadow: none;
 
+  // Token Select
+  --ergo-tokenselect-default-bg: @dark-color-orange-5;
+  --ergo-tokenselect-lbl-color: @dark-color-gray-10;
+
+  --ergo-tokenselect-hover-bg: @dark-color-orange-6;
+  --ergo-tokenselect-active-bg: @dark-color-orange-4;
```

Comment:
The same. Use
--ergo-primary-color-hover
--ergo-primary-color-acitve

Reply:
And why do you use these colors? I suppose the Button component already has that behavior inside.

Reply:
I know it, but I used these colors again in case we use another colors rather than those of Button component.